MEDO v0.5.owl

```diff
@@ -522,12 +671,32 @@ LCME Glossary of Terms for LCME Accreditation Standards and Elements 2015-2016,
     
 
 
+    <!-- http://purl.obolibrary.org/obo//MEDO_0000103 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo//MEDO_0000103">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/MEDO_0000101"/>
+        <obo:IAO_0000115>A formal process of evaluating the degree to which a student or teacher has achieved one or more learning or skill objectives in order to assess and monitor student learning and to provide immediate feedback for the purpose of informing and improving student and teacher praxis resulting in the determination of an official grade or valuation (e.g. daily and/or weekly quizzes/tests, assignments, projects, homework, etc).</obo:IAO_0000115>
+        <rdfs:label>formal formative assessment</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo//MEDO_0000105 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo//MEDO_0000105">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/MEDO_0000101"/>
+        <obo:IAO_0000115>A formal process of evaluating the degree to which a student or teacher has achieved the intended set of learning or skill objectives at the conclusion of an instructional period resulting in the determination of an official grade or valuation (e.g. unit, quarter, semester, year, course, program).</obo:IAO_0000115>
+        <rdfs:label>formal summative assessment</rdfs:label>
+    </owl:Class>
+    
+
+
     <!-- http://purl.obolibrary.org/obo/medo.owl#MEDO_1000 -->
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#MEDO_1000">
-        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0000011"/>
-        <obo:IAO_0000115>a process of evaluating the degree to which a student has achieved a learning objective.</obo:IAO_0000115>
-        <rdfs:label>assessment method</rdfs:label>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/IAO_0000104"/>
+        <obo:IAO_0000115>A modern education theory informed and standards aligned plan specification for evaluating the degree to which a student has achieved one or more learning or skill objectives.</obo:IAO_0000115>
```

Comment:
I think that there can be education assessment methods that are not "modern education theory informed and standards aligned' but still valid. I think the definition should be simplified to "A plan specification for evaluating the degree to which a student has achieved one or more learning or skill objectives."